IDL/inheritance.idl

```diff
@@ -16,6 +16,11 @@ struct InnerStructureHelperEmptyChild : InnerStructureHelper
 {
 };
 
+struct InnerStructureHelperEmptyChildChild : InnerStructureHelperEmptyChild
```

Comment:
Following how the other structures were used, please add this structure as a member in `StructuresInheritanceStruct`. Also add `StructAliasInheritanceStruct` because was forgotten.

Reply:
These bitsets are already included in struct BitsetsChildInheritanceStruct just below :sweat_smile: